ui/controls/radial-chart.reel/_radial-chart.css

```diff
@@ -0,0 +1,36 @@
+.RadialChart {
```

Comment:
@thibaultzanini this is ready for a checkpoint PR to be done.

Spoke to @pchaussalet and he agreed that I should at least have this much merged in for now.